backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -513,6 +529,8 @@ def _post_process_stats(
         )
         effects[is_colinear] = np.nan
         pvals[is_colinear] = np.nan
+        pvals[:, np.all(np.isnan(effects), axis=0)] = 1
+        effects[:, np.all(np.isnan(effects), axis=0)] = 0
```

Comment:
For all-nan columns, just set effect size and p-values to 0 to avoid numpy warnings.